### ISSUE TITLE
Breakout and style the share pop-up window

### ISSUE THREAD

**Issue by codysherman** [codysherman/2NFM#12]
The share dialog needs to be styled, but it's currently handled inline from the javascript. It should be broken into a separate file. The breakout is blocked by #11 first.

**Comment by codysherman** [comment]
Optionally, can we replace the pop-up window completely with just the dropdown.html? Blocked by #35

**Comment by codysherman** [comment]
This pop-up is no longer needed in the new switch to extension-less.
